docs/en/operations/system-tables/dns_cache.md

```diff
@@ -0,0 +1,34 @@
+---
+slug: /en/operations/system-tables/dns_cache
+---
+# dns_cache
+
+Contains information about cached DNS records.
+
+Columns:
+
+- `hostname` ([String](../../sql-reference/data-types/string.md)) — cached hostname
+- `ip_address` ([String](../../sql-reference/data-types/string.md)) — ip address for the hostname
+- `family` ([String](../../sql-reference/data-types/string.md)) — family of the ip address: `IPv4`, `IPv6`, or `UNIX_LOCAL`.
```

Comment:
This should really be of type [Enum](https://clickhouse.com/docs/en/sql-reference/data-types/enum).

Alternatively, replace `ip_address` by `ipv4` (of type `Nullable([IP4](https://clickhouse.com/docs/en/sql-reference/data-types/ipv4))`, `ipv6` (of type `Nulalble(IP6)`) and `unix_local` (of type `String`)

Reply:
thx, it's done